CHANGELOG.md

```diff
@@ -23,6 +23,7 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.1.0/).
 - Add the `SitterInfoCardView` - [#112](https://github.com/ios-course/ironfoudation-team-project/pull/158)
 - Create `PurpleCapsuleOfInfinityWidth` as `ButtonStyle`. Create `LoginView` and apply in the `WoofApp` to display when running the app when the user role is not selected - [#160](https://github.com/ios-course/ironfoudation-team-project/pull/160)
 - Add the `EditSitterInformationView` and the reusable element `TextEditorWithPlaceholder` - [#159](https://github.com/ios-course/ironfoudation-team-project/pull/159)
+- Add `SitterProfileViewModel` for `SitterProfileView` - [#164](https://github.com/ios-course/ironfoudation-team-project/pull/164)
```

Comment:
Where can I find `SitterProfileView`? 
Check the description of the PR.

Reply:
Should I specified it? The name of the file indicates, that this file will be in the View directory, we never specified it.